src/Models/Constants/TelemetryConstants.ts

```diff
@@ -188,26 +188,39 @@ export const TelemetryEvents = {
                 ParseDataHistory: {
                     eventName: 'ParseConfig.DataHistoryExplorerKpis',
                     properties: {
+                        modalId: 'modalId',
                         countSeries: 'countSeries'
                     }
                 }
             },
             UserAction: {
                 /** opening modal to explore data history */
                 OpenModal: {
-                    eventName: 'DataHistoryExplorer.AddSeries'
+                    eventName: 'DataHistoryExplorer.OpenModal',
+                    properties: {
```

Comment:
I would argue that we don't need to define this particular property on every separate event. We can simply have it as a general constant and then attach it to every event.

Reply:
where are you thinking to create that constant then? Since I thought that id should be created when we have the modal component it should be uniquely created in that component. If I am not going to pass it as property where it should be created? isn't having a general constant conflict with your initial intention to tie the modal with a particular interaction session?

Reply:
Okay I made a recent change as far as I understood from your comments, see if that is what you referring

Reply:
![image](https://user-images.githubusercontent.com/45217314/219460951-5aa59c89-3ed0-4aa2-995d-ef71d37ae66d.png)